packages/ibm-products/src/components/Datagrid/Datagrid/addons/Filtering/FilterFlyout.js

```diff
@@ -154,6 +133,29 @@ const FilterFlyout = ({
     );
   };
 
+  /** Effects */
+  useClickOutside(filterFlyoutRef, (target) => {
+    const hasClickedOnDatePicker = target.closest('.flatpickr-calendar');
+    const hasClickedOnDropdown =
+      target.className === 'bx--list-box__menu-item__option';
```

Comment:
I would avoid hard coding the Carbon prefix...
```jsx
import { carbon } from '../../settings';
```

...
...
target.className === `${carbon.prefix}--list-box__menu-item__option`;

Reply:
Good catch!